models/qrCodeAuth.js

```diff
@@ -0,0 +1,53 @@
+const firestore = require("../utils/firestore");
+const qrAuthModel = firestore.collection("QrCodeAuth");
+const USER_DOES_NOT_EXIST_ERROR = "User does not exist.";
+const userModel = firestore.collection("users");
+
+const updateStatus = async (userId, authStatus = "NOT_INIT") => {
+  try {
+    const authData = await qrAuthModel.doc(userId).get();
+
+    if (!authData.data()) {
+      return {
+        userExists: false,
+      };
+    }
+
+    await qrAuthModel.doc(userId).set({
+      ...authData.data(),
+      authorization_status: authStatus,
+    });
+
+    return {
+      userExists: true,
+      data: {
+        ...authData.data(),
+        authorization_status: authStatus,
+      },
+    };
+  } catch (err) {
+    logger.error("Error in updating auth status", err);
+    throw err;
+  }
+};
+
+const storeUserDeviceInfo = async (userDeviceInfoData) => {
```

Comment:
It seems like we are using this model function only for tests. Are we going to use it elsewhere as well for a route?
If not I would suggest moving it to the utils folder of test

Reply:
This function is part of @isVivek99 's PR for post call. I just included this here for testing